### DISCUSSION TITLE
How can it output the partition number of each element directly ?

### DISCUSSION THREAD

**Discussion by Gxming2024** [inducer/pymetis#75] [Q&A]
We can see that Pymetis output the partition number of each node, not the number of each element.  Can I get the partition number of each element directly? How will I do ?

**Comment by inducer** [comment]
Read the metis docs, learn about dual graphs.

**Reply by Gxming2024** [reply]
do you know the method ? Could you tell me directly or remind me?

**Reply by Gxming2024** [reply]
> Read the metis docs, learn about dual graphs.

IF the three nodes of a triangle have different partition number, such as  0, 1, 2. In such case, how can I set the partition number of this triangle ?
